src/main/scala/com/codacy/clangtidy/ClangTidyReportParser.scala

```diff
@@ -0,0 +1,24 @@
+package com.codacy.clangtidy
+
+import java.nio.file.Paths
+
+object ClangTidyReportParser {
+
+  private val ResultRegex =
+    "(.+|[a-zA-Z]:\\\\.+):([0-9]+):([0-9]+): ([^:]+): (.+) \\[(.+?)\\]".r
+
+  def parse(lines: Seq[String]): Seq[ClangTidyResult] = {
+    lines.collect {
+      case result @ ResultRegex(pathStr, line, column, level, txt, checksList) =>
+        val path = Paths.get(pathStr)
+        val check =
+          checksList
+            .split(",")
+            .headOption
+            .getOrElse(throw new Exception(s"Result does not contain checks: '$result''"))
```

Comment:
Isn't this too aggressive?

Reply:
Do you have another suggestion? Just ignore the result?

Reply:
Usually we ignore, but honestly would be nice to have a decision on the process.
In general I prefer this approach but would be could to have the tools using the same strategy.